src/main/app/case/to-api-format.test.ts

```diff
@@ -148,6 +153,10 @@ describe('to-api-format', () => {
       applicant2NameChangedHow: [],
       applicant1CannotUploadSupportingDocument: [DocumentType.NAME_CHANGE_EVIDENCE],
       applicant2CannotUploadSupportingDocument: [DocumentType.NAME_CHANGE_EVIDENCE],
+      applicant1FinancialOrder: YesOrNo.YES,
+      applicant1FinancialOrderFor: [FinancialOrderFor.APPLICANT, FinancialOrderFor.CHILDREN],
+      applicant2FinancialOrder: YesOrNo.NO,
+      applicant2FinancialOrderFor: [],
```

Comment:
just for coverage sake it's worth having a test in 'results' with applicant1FinancialOrder: YesOrNo.NO, and applicant1FinancialOrderFor: [FinancialOrderFor.APPLICANT, FinancialOrderFor.CHILDREN] to test the unsetting of the applicant1FinancialOrderFor field

and same for app2 fields

Reply:
done